api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.proto

```diff
@@ -22,13 +22,17 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // how the endpoint weights are determined. In the ROUND_ROBIN policy,
 // the endpoint weights are sent by the control plane via EDS. However,
 // in this policy, the endpoint weights are instead determined via
-// qps and CPU utilization metrics sent by the endpoint using the Open
-// Request Cost Aggregation (ORCA) protocol. The weight of a given endpoint
-// is computed as qps / cpu_utilization.
+// qps (queries per second), eps (errors per second), and CPU utilization
+// metrics sent by the endpoint using the Open Request Cost Aggregation (ORCA)
+// protocol. A query counts towards qps when successful, otherwise towards both
+// qps and eps. What counts as an error is up to the application to define.
```

Comment:
What is an application?

Reply:
The endpoint, re-worded to use the same term. This policy leaves the rest (period, collection method, definition) to the endpoint other than the protocol.